build/docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM mcr.microsoft.com/dotnet/sdk:5.0.102-1-alpine3.12@sha256:fb1a43b50c7047e5f28e309268a8f5425abc9cb852124f6828dcb0e4f859a4a1 AS build
+FROM mcr.microsoft.com/dotnet/core/sdk:5.0.103-bionic AS build
```

Comment:
This image doesn't exists. 
mcr.microsoft.com/dotnet/sdk:5.0.103-alpine3.12 seems the valid image for 5.0.103.

Reply:
Ah, thank you I was looking for this. For my understanding, where can I find the path to the valid image in the future?

Reply:
Somewhere here:
https://hub.docker.com/_/microsoft-dotnet-sdk/

you can find other images on this page: https://hub.docker.com/_/microsoft-dotnet/
(asp.net or runtime)

Reply:
Also if you looking for ubuntu version (since you tried bionic which is 18.04 ubuntu version) you can try to use `5.0.103-focal` instead of alpine.

Reply:
Fixed in 1d5be02.